src/js/components/Button/Button.js

```diff
@@ -169,6 +171,20 @@ const Button = forwardRef(
       );
     }
 
+    const sendAnalytics = useAnalytics();
+
+    const onClick = useCallback(
+      (event) => {
+        sendAnalytics({
```

Comment:
Would it be at all helpful to automatically include some key information, such as `label` and `id` so the recipient doesn't need to introspect into the DOM to derive them?

Reply:
I originally did that although I wanted access to the dom element anyway so they could put their analytics path in a `data-` attribute and it's pretty trivial to do `event.element.id` compared to `event.id`. We can certainly add the id if it exists though for convenience. I have more of a concern of putting the label there for button since I'm afraid they'll pass it to adobe or amplitude and accidentally cause issues with their data once they localize (I ran into this with OneView when we defaulted some analytics keys to labels. But it would probably be nice to see it in the events